app/Database/Models/TaskLog.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace MyBB\Core\Database\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class TaskLog extends Model
+{
+    /**
+     * The database table used by the model.
+     *
+     * @var string
+     */
+    protected $table = 'task_logs';
+
+    /**
+     * @var bool
+     */
+    public $timestamps = false;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'content',
+    ];
+
+    /**
+     * @return void
+     */
+    public static function boot()
+    {
+        parent::boot();
+
+        static::creating(function ($model) {
+            $model->created_at = $model->freshTimestamp();
+        });
+    }
+
+    /**
+     * Get the task record associated with the log.
+     */
+    public function task()
+    {
+        return $this->hasOne(Task::class, 'id', 'task_id');
```

Comment:
Nice one, thanks. I think we're good to go on merging this then, right? If there's anything else needed it can always be added in another PR.